scripts/ci/pingora_edge_policy.py

```
@@ -0,0 +1,345 @@
+    """Read one bounded GitHub REST JSON document using bearer authentication."""
+
+    _validate_github_api_url(url)
+    request = Request(
+        url,
+        headers={
+            "Accept": "application/vnd.github+json",
+            "Authorization": f"Bearer {token}",
+            "X-GitHub-Api-Version": "2022-11-28",
+            "User-Agent": "cwl-pingora-edge-policy/1",
+        },
+    )
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)